src/els_db.erl

```diff
@@ -2,11 +2,13 @@
 
 %% API
 -export([ find/2
+        , find_multi/2
         , keys/1
         , list/1
         , store/3
         , update/4
         , delete/2
+        , flush_all_tables/0
```

Comment:
This function is only used in tests, but I wonder if it is really needed since the `uri` used in `flush_references_db` test is defined by us and could be something that does not coincide with any of the other files used in the tests.

Reply:
True. It will slightly complicate the tests though, because `flush_references_db` test uses both

* `els_db:list(modules)`
* `els_db:list(references)`

which are otherwise full of random other things.

Reply:
I was thinking that `?assertEqual([{foo,<<"file://tmp/foo.erl">>}],els_db:list(modules))` is maybe not so related to the test and `els_db:list(references)` could be replaced with a call to the new `els_db:find_multi/1`.